lib/hydra/works/services/determine_mime_type.rb

```diff
@@ -0,0 +1,24 @@
+module Hydra::Works
+  class DetermineMimeType
+    # Determines the mime type for a given file
+    # @param [IO, File, Rack::Multipart::UploadedFile, #read] file
+    # @param [String, NilClass] original_name of the file
+    # @return [String]
+    class << self
```

Comment:
At the very minimum, it might be nice to arrive at two patterns:

* a Module-like pattern (a bunch of methods defined on `self`)
* a Class-like pattern (a single method defined on `self` named `#run` or `#call` that instantiates the class, uses attrs to encapsulate state, and assigns passed in params to ivars in `#initialize`) -- and it'd be great to standardize on either `#run` or `#call`